CHANGELOG.md

```diff
@@ -1,3 +1,7 @@
+## 0.12.0 (June 07, 2021)
+
+Integrating the support for the Certificate Policy Management provided by VCert.
```

Comment:
is just:  policy management:

Integrating the support for Policy Management provided by VCert.